resources/client/cl_call.ts

```diff
@@ -1,10 +1,11 @@
 import events from '../utils/events';
 import { ICall } from '../../phone/src/common/typings/call';
+import {phoneCallStartAnim, phoneCallEndAnim} from './functions';
 
 const exp = (global as any).exports;
 
-RegisterNuiCallbackType(events.PHONE_INITIALIZE_CALL);
-on(`__cfx_nui:${events.PHONE_INITIALIZE_CALL}`, (data: any, cb: Function) => {
+RegisterNuiCallbackType(events.PHONE_INITIALIZE_CALL); // Fires when the call is started.
+on(`__cfx_nui:${events.PHONE_INITIALIZE_CALL}`, async (data: any, cb: Function) => {
```

Comment:
This shouldn't need to be async
```suggestion
on(`__cfx_nui:${events.PHONE_INITIALIZE_CALL}`, (data: any, cb: Function) => {
```